web/.vscode/settings.json

```
@@ -1,5 +1,5 @@
 {
-  "editor.codeActionsOnSave": ["source.addMissingImports", "source.fixAll", "source.organizeImports"],
+  "editor.codeActionsOnSave": ["source.addMissingImports", "source.fixAll.eslint", "source.organizeImports"],
```

Changing this as it's currently removing unused imports. This is very annoying when adding an import and saving before you have used it :D
```suggestion
  "editor.codeActionsOnSave": ["source.addMissingImports", "source.fixAll.eslint", "source.sortImports"],
```

---

Man that would be terrible DX with auto save on... 😅

---

Oh yes, it really is - I disabled it instantly locally, but forgot to push the change in any of the branches 😅